### PR TITLE
feat: getSelectionRange API

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

* `getSelectionRange()`
    * return type: `{ start: [number, number], end: [number, number] }`
```
{
 start: [rowIndex, columnIndex],
 end: [rowIndex, columnIndex]
}
```

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: packages/toast-ui.grid/src/grid.tsx [hunk 1]
```diff
@@ -555,6 +555,27 @@ export default class Grid {
...
+   * @returns {Object | null} range - Selection range
+   *     @returns {Array} [range.start] - Index info of start selection (ex: [rowIndex, columnIndex])
+   *     @returns {Array} [range.end] - Index info of end selection (ex: [rowIndex, columnIndex])
+   */
+  public getSelectionRange() {
+    const { rangeWithRowHeader } = this.store.selection;
+
+    if (rangeWithRowHeader) {
+      const { column, row } = rangeWithRowHeader;
+      console.log(rangeWithRowHeader);
```
Comment (dongwoo-kim) [inline packages/toast-ui.grid/src/grid.tsx]: console.log 제거요~ ㅎ

File: packages/toast-ui.grid/cypress/integration/selection.spec.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    const { document, tui } = win;
+    const options = { data, columns, ...customOptions };
+    const el = document.createElement('div');
+    el.style.width = `${CONTENT_WIDTH + SCROLLBAR_WIDTH}px`;
+    document.body.appendChild(el);
+
+    win.grid = new tui!.Grid({ el, ...options });
+    cy.wait(10);
+  });
+}
```
Comment (dongwoo-kim) [inline packages/toast-ui.grid/cypress/integration/selection.spec.ts]: 특별한 이유가 없으면 `cy.createGrid()`를 활용하는 게 좋을 것 같은데..
굳이 `createGrid()`를 따로 만드는 이유가 있나요?

Comment (js87zz) [reply packages/toast-ui.grid/cypress/integration/selection.spec.ts]: 스토리북 분리작업하고 테스트 추가 작업할때 이 부분도 한번 다 정리해야겠네요

Comment (jung-han) [reply packages/toast-ui.grid/cypress/integration/selection.spec.ts]: 기존함수 제거해서 `cy.createGrid()`로 변경했습니다!

**Pull request merged**